src/config.ml

```diff
@@ -625,11 +635,19 @@ let a_x509_cert_payload ctx constructor str =
   | Error (`Msg msg) -> Error (Fmt.str "%s: invalid certificate: %s" ctx msg)
```

Comment:
the `a_x509_cert_payload` is now unused!?

Reply:
Yes, it was no longer possible to share the code. I removed `a_x509_cert_payload` in a commit I just pushed.